common/src/main/java/com/wynntils/features/chat/BombBellRelayFeature.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.chat;
+
+import com.wynntils.core.components.Handlers;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.consumers.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyBind;
+import com.wynntils.models.worlds.type.BombInfo;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import org.lwjgl.glfw.GLFW;
+
+public class BombBellRelayFeature extends Feature {
+    @RegisterKeyBind
+    private final KeyBind relayPartyKeybind =
+            new KeyBind("Relay Bomb to Party", GLFW.GLFW_KEY_UNKNOWN, true, this::relayToParty);
+
+    @RegisterKeyBind
+    private final KeyBind relayGuildKeybind =
+            new KeyBind("Relay Bomb to Guild", GLFW.GLFW_KEY_UNKNOWN, true, this::relayToGuild);
+
+    private String getAndFormatLastBomb() {
+        BombInfo lastBomb = Models.Bomb.getLastBomb();
+        if (lastBomb == null) return null;
+
+        // Intentionally not localized as the majority of the playerbase speaks English
```

Comment:
I don't think that makes sense. We mostly have English translations, but we always use i18n hooks. Remember that you can use placeholders for variables in the i18n texts, so this is not any more tricky to translate than any other text.

Reply:
But, let's say I am german, I play with German language, but I am in an English guild. Obviously I want to send the message as english.

Reply:
Ah, I see. Right, this is the message sent to other players. Hm. Do we have any other places where we sent out chat messages like this? Are they kept in English, or are they mistakenly translated? Like coordinate sharing etc.

Reply:
They are all hard coded afaik.

Reply:
Coordinate ones look okay 
![image](https://github.com/Wynntils/Artemis/assets/57310593/8a6da0d4-4428-4b3b-b95b-521150d9f68f)

Reply:
Ok, good. Maybe we should add a comment at those places too, to describe why they are not localized.